skimage/transform/hough_transform.py

```diff
@@ -151,9 +146,18 @@ def hough_ellipse(image, threshold=4, accuracy=1, min_size=4, max_size=None):
 
     Notes
     -----
-    The accuracy must be chosen to produce a peak in the accumulator
-    distribution. In other words, a flat accumulator distribution with low
-    values may be caused by a too low bin size.
+
```

Comment:
I think this extra line may cause an issue in doc rendering in Sphinx (it used to be like that in the past). Please, remove.